src/api/retunableMediaElement.ts

```diff
@@ -0,0 +1,15 @@
+import { RetunableMediaElementSettings } from "components/attendee/RetunableMediaElement/types";
+import firebase from "firebase/compat/app";
+
+import { COLLECTION_RETUNABLE_MEDIA_ELEMENTS } from "settings";
+
+export const setRetunableMediaSettings = async (
+  spaceId: string,
+  settings: RetunableMediaElementSettings
+) => {
```

Comment:
This might help for more uniformed interface with other functions that try to make it easier to expand with new options and/or ignore old ones

```suggestion

type SetRetunableMediaSettings = ( options: {
  spaceId: SpaceId,
  settings: RetunableMediaElementSettings
}) => Promise<void>;

export const setRetunableMediaSettings: SetRetunableMediaSettings  = async ({
  spaceId,
  settings,
}) => {
```

Reply:
Resolved in 40c9750b0